app/scripts/views/ready.js

```diff
@@ -40,10 +42,21 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
 
     context: function () {
       var serviceName = this.relier.get('serviceName');
+      var redirectUri = this.relier.get('redirectUri');
+      var verificationRedirect = this.relier.get('verificationRedirect');
+      var showProceedButton = false;
+
+      // if this relier wants to use verification redirect in the same browser then create a link to the service
+      if (verificationRedirect === Constants.VERIFICATION_REDIRECT_ALWAYS &&
```

Comment:
Perhaps to avoid last minute confusion and [unexpected behavior for reset password](https://github.com/mozilla/fxa-content-server/pull/2366#issuecomment-101411879), you should add `&& this.is('sign_up')`. We can iterate on the other cases next train.